omod/src/main/java/org/openmrs/module/fhir2/providers/ObservationFhirResourceProvider.java

```diff
@@ -61,8 +64,15 @@ public Observation getObservationById(@IdParam @NotNull IdType id) {
 	public Bundle searchObservations(@OptionalParam(name = Observation.SP_ENCOUNTER) ReferenceParam encounterReference,
 	        @OptionalParam(name = Observation.SP_SUBJECT, chainWhitelist = { "", Patient.SP_IDENTIFIER, Patient.SP_GIVEN,
 	                Patient.SP_FAMILY, Patient.SP_NAME }, targetTypes = Patient.class) ReferenceParam patientReference,
+	        @OptionalParam(name = Observation.SP_HAS_MEMBER) ReferenceParam hasMemberReference,
+	        @OptionalParam(name = Observation.SP_VALUE_CONCEPT) TokenAndListParam valueConcept,
+	        @OptionalParam(name = Observation.SP_VALUE_DATE) DateRangeParam valueDateParam,
+	        @OptionalParam(name = Observation.SP_VALUE_QUANTITY) QuantityAndListParam valueQuantityParam,
+	        @OptionalParam(name = Observation.SP_VALUE_STRING) StringAndListParam valueStringParam,
+	        @OptionalParam(name = Observation.SP_DATE) DateRangeParam date,
 	        @OptionalParam(name = Observation.SP_CODE) TokenAndListParam code, @Sort SortSpec sort) {
```

Comment:
I'd actually prefer it if `SP_CODE` came after the `SP_SUBJECT` and before all these other search functions.

Reply:
will make this change in the next commit